pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go

```diff
@@ -4044,62 +4076,6 @@ var _ = Describe("Test Application Controller", func() {
 		}))
 	})
 
-	It("app record execution state with controllerRevision", func() {
```

Comment:
is this test case deleted?

Reply:
Yes, application revision will replace controller revision and the logic of controller revision is deleted.